packages/codemods/src/modRunner/modFilter.ts

```diff
@@ -0,0 +1,33 @@
+import { CodeMod } from '../codeMods/types';
+import { Maybe } from '../maybe';
+
+export function getStringFilter(filter: string) {
+  const filterString = filter;
+  return (modString: string) => {
+    return modString === filterString;
+  };
+}
+
+export function getRegexFilter(filter: string) {
+  const regexFilter = new RegExp(filter);
+  return (modString: string) => {
+    return regexFilter.test(modString);
+  };
+}
+
+/**
+ * Returns a function that will return true if the mod is found in the filters
+ */
+export function getModFilter<T>(filters: { stringFilter: Maybe<string[]>; regexFilter: Maybe<string[]> }) {
+  let filts: ((modString: string) => boolean)[] = [];
+  filts = filts.concat(filters.stringFilter.then(v => v.map(getStringFilter)).orElse([]));
+  filts = filts.concat(filters.regexFilter.then(v => v.map(getRegexFilter)).orElse([]));
+
+  // Return default filter if no other filters provided
+  if (filts.length === 0) {
+    return (mod: CodeMod<T>) => true;
+  }
+  return (mod: CodeMod<T>) => {
+    return filts.some(v => v(mod.name));
```

Comment:
As a consumer, I would expect this to be `all` instead of `some`.

Reply:
I'll change it to have includeMods and excludeMods and have them be mutually exclusive to clear that up.